supply-chain.md

```diff
@@ -124,41 +122,33 @@ If we successfully added "CertifyBad", the output will show:
-   that we created earlier:
-
-      ```bash
-      Use the arrow keys to navigate: ↓ ↑ → ←
-      ? Select CertifyBad to Query:
-      ↑   pkg:golang/github.com/kr/pretty (pretty bad undisclosed vuln)
-          git+https://github.com/googleapis/google-cloud-go (github repo compromised)
-          pkg:golang/github.com/pmezard/go-difflib (github repo compromised)
-        ▸ pkg:maven/org.apache.logging.log4j/log4j-core@2.8.1 (never use this version of log4j)
-         pkg:golang/github.com/prometheus/client_golang@v1.11.1 (undisclosed vuln)
-      ```
+   `pkg:deb/debian/tzdata@2021a-1+deb11u5 (compromised version of tzdata)`
+   that we created earlier.
 
      Doing so will produce a output similar to this:
 
       ```bash
-      ✔ pkg:maven/org.apache.logging.log4j/log4j-core (never use this version of log4j)
-      Visualizer url: http://localhost:3000/?path=142367,15573,15572,15515,2509,15574,15337,15336,15335,2
+      ✔ pkg:deb/debian/tzdata@2021a-1+deb11u5 (compromised version of tzdata)
+      Visualizer url: http://localhost:3000/?path=142605,44614,1372,1305,1304,127547,127527,127526,36248,2,125455,125358,125357,123291,123287,123286,121220,121216,121215,119149,119145,119144,117075,117074,117073,115010,115006,115005,112939,112935,112934,110299,110283,110282,107515,107453,107452,68077,67990,67989,65923,65745,65744,63678,63674,63673,61607,61603,61602,59536,59532,59531,57463,57461,57460,55393,55390,55389,53320,53319,53318,51236,51113,51112,49048,48779,48778,46714,46713,46712,44615,44610,44609,42533,42528,42527,40466,40462,40461,38397,38393,38392,36252,36250,36249,15392,15337,15336,15335,4155,4125,4124,3,3182,2865,2864,2667,2633,2632,2501,2419,2418,2413,2312,2311,2190,2150,2149,2092,2048,2047,1374,1303,1302
       ```
 
 3. Navigate to the URL to visualize the output. 
-   This will show an expanded graph of dependencies. 
-
-   **If you don't see dependencies expanded here, or less than what is shown below**, re-run the CLI in a few minutes. Additional open-source   
-   insights might still be being ingested from GUAC's external sources (such as Deps.dev).
+   This will show an expanded graph of dependencies.
 
    ![An image of the visualizer output graph](assets/images/supplychain_dependencies_graph.png)
 
-   We can tell from this small example (arranging the graph a little) that
-   the bad package is being used by a test image under the guacsec org! We need to
-   remediate that right away!
+   We can tell from this example (arranging the graph a little) the bad
+   debian package (used for timezone information) is commonly used throughout a
+   bunch of dependant container images! All are a cause for concern as they are
+   notable images for kubernetes, redis, nginx and python. We need to remediate
+   these right away! This allows us to quickly figure out what needs to be updated,
+   so we are not scrambling to first scan and determine where `tzdata` might be
+   used.
```

Comment:
@sarahlinnl to review this change

Reply:
Capitalize proper nouns like Python.
"so we are not scrambling" > "so that we are not scrambling"